generics/helpers/elasticSearch.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * name : elasticSearch.js
+ * author : Aman Jung Karki
+ * created-date : 11-Jun-2020
+ * Description : elastic search common functionality
+ */
+
+/**
+  * Get user data
+  * @function
+  * @name get
+  * @param {Object} data
+  * @param {String} data.id - user id
```

Comment:
@aman-tunerlabs - It could be any id right, I mean in general this function, it's not just for users so, the description can be "a unique document id"

I would suggest having different arguements for index, type and id, not all in variable data